tests/suites/test_suite_ssl.function

```diff
@@ -2597,6 +2597,25 @@ int tweak_tls13_certificate_msg_vector_len(
                                         pwd_string, pwd_len );  \
     TEST_EQUAL( ret, exp_ret_val )
 #endif
+
+#define TEST_AVAILABLE_ECC( _tls_id, _group_id, _psa_family, _psa_bits ) \
```

Comment:
I'm not sure if we are using `_` notation in this project. I remember in one of my first PRs I tried to use this and for some reason it was rejected, but can't find it now.

Can we fix formatting so that each `\` is in the same column?

Reply:
I don't remember the details either, but indeed we should not use C identifiers that that with `_`. However here this will only be seen by the pre-processor, so I think it's fine. However, perhaps it's better to use names that don't start with `_` just for the sake of uniformity and to avoid confusion. (I think names ending with `_` are fine though.)

Reply:
[In C](https://en.cppreference.com/w/c/language/identifier), identifiers that start with an underscore and a lowercase letter are reserved in the global namespace, but ok as local identifiers. Identifiers that start with an underscore and an uppercase letter or with two underscores are reserved everywhere. [In C++](https://en.cppreference.com/w/cpp/language/identifiers), which matters to us only in public headers, identifiers with two underscores anywhere in the name are also reserved everywhere.

“Never start identifiers with underscores” is a safe, simple approximation.